src/installing-workflow/system-requirements.md

```diff
@@ -26,12 +26,9 @@ application footprint as well.
 Running smaller machines will likely result in increased system load and has been known to result in component failures
 and instability.
 
-## Docker Insecure Registry
-
-The on-cluster Docker registry is not deployed with TLS enabled. As such, all Kubernetes worker nodes must have their
-Docker daemons configured to use an **insecure registry**. The configured subnet should encompass any private networks
-used by your worker nodes, including overlay networks.
+!!! warning
 
+    Versions prior to 2.2 require '--insecure-registry' to function properly.
```

Comment:
This needs an extra newline character to signify the end of the admonition and the beginning of the paragraph, or else the next line will be considered part of the admonition.